packages/blockchain-wallet-v4-frontend/src/data/modules/profile/sagas.ts

```diff
@@ -554,6 +565,7 @@ export default ({ api, coreSagas, networks }) => {
     syncUserWithWallet,
     updateUser,
     updateUserAddress,
-    waitForUserData
+    waitForUserData,
+    waitForUserId
```

Comment:
Let's not export waitForUserId, let's only expose waitForUserData. In analytics saga lets also just waitForUserData